### PR TITLE
SPUI-150 > Enable email verification flow

### PR DESCRIPTION
**Pull request by eli1stark**

### CODE REVIEW

File: lib/auth/screens/email_verification/notifier/email_verification_notifier.dart [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          ),
+        );
+
+  final String email;
+  final ValidationService service;
+  final BuildContext context;
+
+  static final _logger = Logger('EmailVerificationNotifier');
+
+  // void updateCode(String? code) {
```
Comment (VovaMedvediev) [inline lib/auth/screens/email_verification/notifier/email_verification_notifier.dart]: do we need this commented code?

File: lib/auth/screens/email_verification_success/email_verification_success.dart [hunk 2]
```diff
@@ -39,21 +43,23 @@ class EmailVerificationSuccess extends HookWidget {
               Expanded(
                 child: Column(
                   mainAxisAlignment: MainAxisAlignment.center,
-                  children: const [
-                    Spacer(),
-                    SizedBox(
+                  children: [
+                    const Spacer(),
+                    const SizedBox(
```
Comment (VovaMedvediev) [inline lib/auth/screens/email_verification_success/email_verification_success.dart:48]: For what reason do we need SizedBox here? It seems that it is not visible on the screen

File: lib/shared/services/remote_config_service.dart [hunk 3]
```diff
@@ -7,8 +7,9 @@ class RemoteConfigService {
   Future<void> overrideBaseUrls() async {
     await _config.fetchAndActivate();
 
-    tradingAuthBaseUrl = _config.getString('tradingAuthBaseUrl');
-    walletApiBaseUrl = _config.getString('walletApiBaseUrl');
-    tradingBaseUrl = _config.getString('tradingBaseUrl');
+    tradingAuthApi = _config.getString('tradingAuthApi');
```
Comment (VovaMedvediev) [inline lib/shared/services/remote_config_service.dart:10]: This is the reason of broken firebase config. It is needed to rename key name (e.g rename tradingAuthBaseUrl to tradingAuthApi)  in the firebase console. Or don't change key name at all.

File: lib/main.dart [hunk 4]
```diff
@@ -31,7 +30,7 @@ Future<void> main() async {
   await Firebase.initializeApp();
   if (!kIsWeb) {
     await PushNotificationService().initialize();
-    await RemoteConfigService().overrideBaseUrls();
+    // await RemoteConfigService().overrideBaseUrls();
```
Comment (VovaMedvediev) [inline lib/main.dart:33]: Uncomment it please after fix. (see last comment)

**Review by VovaMedvediev — CHANGES_REQUESTED** [review]

File: lib/auth/screens/email_verification/notifier/email_verification_notifier.dart (hunk 1 above)
Comment (eli1stark) [reply lib/auth/screens/email_verification/notifier/email_verification_notifier.dart]: no, deleted

File: lib/auth/screens/email_verification_success/email_verification_success.dart (hunk 2 above)
Comment (eli1stark) [reply lib/auth/screens/email_verification_success/email_verification_success.dart:48]: In order to stretch column to max size

Comment (eli1stark) [reply lib/auth/screens/email_verification_success/email_verification_success.dart:48]: Actually it's not needed anymore I deleted it

File: lib/shared/services/remote_config_service.dart (hunk 3 above)
Comment (eli1stark) [reply lib/shared/services/remote_config_service.dart:10]: Probably it was some delay on the Firebase's side, now remote config is working

File: lib/main.dart (hunk 4 above)
Comment (eli1stark) [reply lib/main.dart:33]: Uncommented

**Pull request merged**